forte/base_classes/coupling_coefficients.h

```diff
@@ -0,0 +1,61 @@
+#ifndef _coupling_coefficients_h_
+#define _coupling_coefficients_h_
+
+#include <ambit/tensor.h>
+
+namespace forte {
+
+class CICouplingCoefficients {
```

Comment:
Please document what coupling coefficients _are_.

Reply:
Do you want a real documentation or simply comment the code?

Reply:
The index convention is actually documented in the function that computes these CCs. I can move it here but I am not sure if this container is responsible for that.

Coupling coefficients are simply the elements without summing over integral indices when computing CI matrix.

Reply:
Oh! Simply saying "the caller defines the indexing conventions" would be good.